readme.txt

```diff
@@ -93,6 +93,22 @@ Would you like to help translate the plugin into more languages? [Join our WP-Tr
 
 == Frequently Asked Questions ==
 
+= Where can I find complete documentation? =
+
+On our web site in the top right you can find [our support page](http://docs.easydigitaldownloads.com/?utm_source=docs&utm_medium=installation_tab&utm_content=documentation&utm_campaign=readme)
```

Comment:
No need to describe the location of the link if linking directly to the page. I'd prefer we not describe the actual position of the link since that makes it harder for us to change it if we wish.